src/Stratis.Features.FederatedPeg/TargetChain/WithdrawalTransactionBuilder.cs

```diff
@@ -77,5 +79,18 @@ public Transaction BuildWithdrawalTransaction(uint256 depositId, uint blockTime,
             this.logger.LogTrace("(-)[FAIL]");
             return null;
         }
+
+        /// <summary>
+        /// We need to reduce the amount being withdrawn by the fees our transaction is going to have.
+        /// </summary>
+        private static Recipient ReduceRecipientPaymentByFee(Money transactionFee, Recipient recipient)
```

Comment:
This could go on `Recipient`.

Reply:
You probably don't need a private method just for doing this.